src/app/layouts/common-layout/common-layout.routing.ts

```diff
@@ -1,12 +1,22 @@
-import {Routes} from '@angular/router';
-import {CompanyComponent} from '../../pages/company/company.component';
-import {AuthGuard} from '../../helpers/auth.guard';
-import { RegistersuccessComponent } from '../../pages/registration/registersuccess/registersuccess.component';
-import { ProfileComponent } from 'src/app/pages/profile/profile.component';
-
+import { Routes } from "@angular/router";
+import { CompanyComponent } from "../../pages/company/company.component";
+import { AuthGuard } from "../../helpers/auth.guard";
+import { RegistersuccessComponent } from "../../pages/registration/registersuccess/registersuccess.component";
+import { ProfileComponent } from "src/app/pages/profile/profile.component";
+import { HomeComponent } from "../../pages/home/home.component";
+import { PropertiesComponent } from "../../pages/properties/properties.component";
+import { NoPropertiesComponent } from "src/app/pages/properties/no-properties/no-properties.component";
 
 export const CommonLayoutRoutes: Routes = [
-  {path: 'create-company', component: CompanyComponent, canActivate: [AuthGuard]},
-  { path: 'registersuccess', component: RegistersuccessComponent },
-  { path: 'profile', component: ProfileComponent }
+  {
+    path: "create-company",
```

Comment:
@Genza999 the url is still not conventional. Could you look into this? Also, since we are working on the frontend, you can read this [article](https://www.clickz.com/importance-of-url-naming-convention-and-url-management-for-websites/37490/)

Reply:
Hey @malep2007 , thankyou for the feedback but this was not part of my feature